Formula/nginx-unit.rb

```diff
@@ -0,0 +1,256 @@
+class NginxUnit < Formula
+  desc "Dynamic web and application server for multiple languages"
+  homepage "https://unit.nginx.org/"
+  url "https://unit.nginx.org/download/unit-1.1.tar.gz"
+  sha256 "7c66365f5ea87e7e8903bac9d639676947fe2ab7aa799ce35defc426b3409ee0"
+
+  depends_on "openssl"
+  depends_on "pcre"
+
+  def install
+    openssl = Formula["openssl"]
+    pcre = Formula["pcre"]
+
+    system "./configure",
+      "--prefix=#{prefix}",
+      "--bindir=#{bin}",
+      "--sbindir=#{bin}",
```

Comment:
Why are we relocating things out of sbin? Does it by default installing some things in bin and others in sbin?

Reply:
It doens't put anything into bin by default, only `unitd` into sbin. But there is no sbin in `/usr/local`, and macOS doesn't have it in $PATH. So I followed the example from the nginx formula and used bin.

See the design behind these paths in an earlier comment:

https://github.com/Homebrew/homebrew-core/pull/27260#discussion_r190568505

Reply:
I'd leave it as sbin.

Reply:
This now fails for me:

```
homebrew-core (unit*) % brew reinstall nginx-unit
==> Reinstalling nginx-unit
==> Downloading https://unit.nginx.org/download/unit-1.1.tar.gz
Already downloaded: /Users/sj26/Library/Caches/Homebrew/nginx-unit-1.1.tar.gz
==> ./configure --prefix=/usr/local/Cellar/nginx-unit/1.1 --modules=/usr/local/lib/unit --state=/usr/
==> make install
==> ./configure perl
==> make perl
==> ./configure python
==> make python
==> ./configure ruby
==> make ruby
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink sbin/unitd
/usr/local/sbin is not writable.

You can try again using:
  brew link nginx-unit
==> Caveats
Once running, you can control unit using the control socket, for example using
curl:
  curl --unix-socket /usr/local/var/run/unit.control.sock http://localhost/

Modules are available for system perl, python and ruby. The system php doesn't
support nginx-unit. You can install additional modules into:
  /usr/local/lib/unit

To have launchd start nginx-unit now and restart at login:
  brew services start nginx-unit
Or, if you don't want/need a background service you can just run:
  /usr/local/sbin/unitd --no-daemon
==> Summary
🍺  /usr/local/Cellar/nginx-unit/1.1: 11 files, 384.1KB, built in 31 seconds
```

Reply:
what's the output of
```
ls -ld /usr/local/sbin
```

Reply:
```
$ ls -ld /usr/local/sbin
ls: /usr/local/sbin: No such file or directory
```

```
$ mkdir /usr/local/sbin
mkdir: /usr/local/sbin: Permission denied
```

Homebrew doesn't make an sbin during installation, and can't without sudo because of SIP, afaik, and doesn't offer the option to do it.

Reply:
We do indeed make that directory https://github.com/Homebrew/install/blob/master/install#L206

you'll need to fix it locally :)

Reply:
Wow! Mea culpa, sorry, I don't know why I thought otherwise. I wonder what happened to mine! :+1: